tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -65,7 +63,7 @@ def test_5_snapshots(
         mem_size_mib=512,
         track_dirty_pages=diff_snapshots,
     )
-    tap, host_ip, guest_ip = vm.ssh_network_config(network_config, "1")
+    vm.add_net_iface()
```

Comment:
`NetIfaceConfig` is unimported but is still used in "[test: clean up network interfaces and SSH helper](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/a8fb4b7369f8ede378399a88cb42adee05b25990)"